### DISCUSSION TITLE
How to replace "find" operator with mixed-integer expressions?

### DISCUSSION THREAD

**Discussion by Lahm919** [yalmip/YALMIP#1452] [Optimization modelling]
When I was using the "find" operator recently, I noticed that the end of the documentation mentioned that you could implement the "find" operator with mixed-integer expressions based on the Big-M method. However, I could not find a detailed description of this mixed-integer alternative method, so I would like to ask you for more information.
Specifically, I wish to know the following:
How do we replace the "find" operator with mixed-integer expressions based on the Big-M method in YALMIP?
Are there any relevant examples or documents for reference?

**Comment by johanlofberg** [comment]
https://yalmip.github.io/command/find/
